modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/resources/app.conf

```diff
@@ -0,0 +1,34 @@
+#
+# Seqexec server configuration for development mode
+#
+
+# mode can be dev in which case fake authentication is supported and the UI provides some extra tools
+mode = "dev"
+
+# Authentication related settings
+authentication {
+    # Indicates how long a session is valid in hrs
+    sessionLifeHrs = 8
+    # Name of the cookie to store the session
+    cookieName = "SeqexecToken"
+    # Secret key for JWT tokens
+    secretKey = "Ofm@uYN5zcRL0qwNlMR75%m70HLG@Rz9INv1Bhv"
```

Comment:
That is a very good concern. In this case we don't care since it is a development/test instance, I have already code to get the "secret" configuration for **production** stored in svn. I didn't want to make this PR more complex than it is